### PR TITLE
Edit availability constraints for `checked_oint` v0.2.0

### PR DESCRIPTION
**Pull request by hirrolot**
See [this discussion](https://github.com/ocaml/opam-repository/pull/26662#discussion_r1787311683).

### CODE REVIEW

**Comment by hirrolot** [comment]
I tried `os-distribution != "opensuse"`, `os-family != "opensuse"`, `os != "opensuse"`, but nothing works to disable building on openSUSE. Any ideas?

**Comment by hirrolot** [comment]
It's interesting that `os-family != "opensuse"` disables `opensuse-tumbleweed` but not `opensuse-15.6`...

File: packages/checked_oint/checked_oint.0.2.0/opam [hunk 1]
```diff
@@ -17,17 +17,7 @@ depends: [
...
-  "fedora-38"
-  "fedora-39"
-  "fedora-40"
-  "opensuse-15.5"
-  "opensuse-tumbleweed"
-  "oraclelinux-8"
-  "oraclelinux-9"
-  "freebsd"
-]
+available: arch != "arm32" & arch != "x86_32" & arch != "ppc64" & os != "win32" & os != "freebsd" & os-distribution != "fedora" & os-distribution != "oraclelinux" & os-family != "opensuse" & switch != "5.3~alpha1"
```
Comment (mseri) [inline packages/checked_oint/checked_oint.0.2.0/opam:20]: You need this
```suggestion
available: arch != "arm32" & arch != "x86_32" & arch != "ppc64" & os != "win32" & os != "freebsd" & os-distribution != "fedora" & os-distribution != "oraclelinux" & os-family != "opensuse" & os-family != "suse"
```

if the library is not compatible with ocaml 5.3 I would suggest an upper bound on the ocaml dependency instead. This is the alpha release of the compiler though, so I thunk it is better not to add any extra compiler constraint for the moment

Comment (mseri) [reply packages/checked_oint/checked_oint.0.2.0/opam:20]: I forgot to mention that the various os flags are documented here https://github.com/ocaml/opam-repository/wiki/Depexts-os-distribution---os-family-values

**Comment by hirrolot** [comment]
Thanks for helping me @mseri, the CI works right now except `5.3~alpha1`.

**Comment by mseri** [comment]
Thanks a lot and sorry for the slow response!

**Pull request merged**